ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java

```diff
@@ -18,7 +22,24 @@
     private static final ExtendedMotor rightFrontCAN = FRC.talonCAN(2), rightBackCAN = FRC.talonCAN(3);
     private static final ExtendedMotor leftFrontCAN = FRC.talonCAN(0), leftBackCAN = FRC.talonCAN(1);
 
+    public static final FloatCell autoLeftMotors = new FloatCell();
+    public static final FloatCell autoRightMotors = new FloatCell();
```

Comment:
Perhaps these should be moved to the autonomous code, and referenced from there. Better locality, perhaps?